### ISSUE TITLE
Solver gets confused when reloading a finished game

### ISSUE THREAD

**Issue by ericpromislow** [ericpromislow/lirdle#29]
Need to track whether we're done or not and not process the last guess if we are finished

**Comment by ericpromislow** [comment]
On an iphone/safari, I finished the game, closed the tab, then reopened it, and have an incomplete gameboard. I can't type anything, but the copy-to-clipboard buttons and stats box aren't present. Probably because the solver is broken. Reproducible in safari:

```
[Log] Error loading local storage: Error: Can't happen - no words match this line (model.js, line 50)
```

**Comment by ericpromislow** [comment]
Fixed in a1d83b7
